out/production/auto-scheduler/org/txstate/auto_scheduler/MANIFEST.MF

```diff
@@ -0,0 +1 @@
+Main-Class: org.txstate.auto_scheduler.Main
```

Comment:
not sure we are adding code file to an out folder. 
this file currently exists under `https://github.com/samaniehsan/auto-scheduler/blob/master/code/src/org/txstate/auto_scheduler/MANIFEST.MF`